x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/connector_edit_flyout.tsx

```diff
@@ -85,6 +85,15 @@ export const ConnectorEditFlyout = ({
     Option<ActionTypeExecutorResult<unknown>>
   >(none);
   const [isExecutingAction, setIsExecutinAction] = useState<boolean>(false);
+  const handleSetTab = useCallback(
+    (newTab) => {
+      setTab(newTab);
+      if (testExecutionResult !== none && newTab !== EditConectorTabs.Test) {
```

Comment:
There was a bug here. We are not clearing the result message (success/fail) even though we clear the form when we navigate off of the Test tab. This checks if theres a result and we're navigating off the tab, then clear the result